content/engineering/languages-runtimes/language-selection.md

```diff
@@ -37,11 +37,28 @@ Looking at this same data and applying preferences from the guiding factors belo
 ## Frequently-used frameworks
 The following are used widely at 18F:
 
-| Purpose  |  Tool  |
-| ---- | ---- |
-| CSS framework  |  [_More info_]({{ "/engineering/languages-runtimes/css/#frameworks" | url }})  |
-| Infrastructure/configuration as code | [Terraform](https://www.terraform.io/)  |  
-| Static site generator  | [Jekyll](https://jekyllrb.com/) (with the [uswds-jekyll](https://github.com/18F/uswds-jekyll) theme) or Hugo |
+<table>
+  <thead>
+    <tr>
+      <th>Purpose</th>
+      <th>Tool</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>CSS framework</td>
+      <td><a href="{{ "/engineering/languages-runtimes/css/#frameworks" | url }}"><em>More info</em></a></td>
+    </tr>
+    <tr>
+      <td>Infrastructure/configuration as code</td>
+      <td><a href="https://www.terraform.io/">Terraform</a></td>
+    </tr>
+    <tr>
+      <td>Static site generator</td>
+      <td>{% include "engineering/tag-default.html" %} <a href="https://www.11ty.dev/">11ty/Eleventy</a></td>
+    </tr>
+  </tbody>
+</table>
```

Comment:
The default tag snippet has a trailing newline, which is pretty standard (our prettier config wants it, for example). However, if you include the tag within markdown, it gets inserted into the markdown before the markdown is parsed, which means it can break markdown syntax. For example, a table row like this:

```
| Static site generator | {% include "tag" %} 11ty |
```

becomes

```
| Static site generator | tag
11ty |
```

When then renders as two table rows instead of one. The fix options are either remove the trailing newline from the tag snippets, which seems very fragile because anyone in the future who tweaks one and runs prettier would suddenly break this table, or replace the markdown table with an HTML one. I chose the latter. 😛